CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 0.1.41
+
+<!-- generated comparing v0.1.40..master -->
+
+_Dec 13, 2023_
+
+Support server-side sorting and filtering, support delete in data providers.
+
+TODO INSERT HIGHLIGHTS
```

Comment:
unintended 🙂

Reply:
oh no

Reply:
i'll add a commit to remove it, thanks for spotting it